doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst

```diff
@@ -533,39 +533,41 @@ The soil moisture deficit :math:`D_{irrig}` is
 
    D_{irrig} = \left\{
    \begin{array}{lr}
-   w_{thresh} - w_{avail} &\qquad w_{thresh} > w_{avail} \\
+   w_{target} - w_{avail} &\qquad w_{thresh} > w_{avail} \\
    0 &\qquad w_{thresh} \le w_{avail}
    \end{array} \right\}
 
-where :math:`w_{thresh}` is the irrigation moisture threshold (mm) and :math:`w_{avail}` is the available moisture (mm). The moisture threshold is
+where :math:`w_{target}` is the irrigation target soil moisture (mm)
 
 .. math::
    :label: 25.62
 
-   w_{thresh} = f_{thresh} \left(w_{target} - w_{wilt}\right) + w_{wilt}
+   w_{target} = \sum_{j=1}^{N_{irr}} \theta_{target} \Delta z_{j} \ .
 
-where :math:`w_{target}` is the irrigation target soil moisture (mm)
+The irrigation moisture threshold (mm) is
 
 .. math::
    :label: 25.63
 
-   w_{target} = \sum_{j=1}^{N_{irr}} \theta_{target} \Delta z_{j} \ ,
+   w_{thresh} = f_{thresh} \left(w_{target} - w_{wilt}\right) + w_{wilt}
 
-:math:`w_{wilt}` is the wilting point soil moisture (mm)
+where :math:`w_{wilt}` is the wilting point soil moisture (mm)
 
 .. math::
    :label: 25.64
 
    w_{wilt} = \sum_{j=1}^{N_{irr}} \theta_{wilt} \Delta z_{j} \ ,
 
 and :math:`f_{thresh}` is a tuning parameter.  The available moisture in
-the soil is
+the soil (mm) is
 
 .. math::
    :label: 25.65
 
    w_{avail} = \sum_{j=1}^{N_{irr}} \theta_{j} \Delta z_{j} \ ,
 
+Note that :math:`w_{target}` is truly supposed to give the target soil moisture value that we're shooting for whenever irrigation happens; then the soil moisture deficit :math:`D_{irrig}` gives the difference between this target value and the current soil moisture. The irrigation moisture threshold :math:`w_{thresh}`, on the other hand, gives a threshold at which we decide to do any irrigation at all. The way this is written allows for the possibility that one may not want to irrigate every time there becomes even a tiny soil moisture deficit. Instead, one may want to wait until the deficit is larger before initiating irrigation; at that point, one doesn't want to just irrigate up to the "threshold" but instead up to the higher "target". The target should always be greater than or equal to the threshold.
```

Comment:
Great explanation!

Reply:
Actually, that was @billsacks 's great explanation.